### ISSUE TITLE
Add a command to negate the exit code of some other command

### ISSUE THREAD

**Issue by Shayan-To** [shelljs/shx#173]
I couldn't find any way to do this in a cross-platform way. POSIX shell has the `!` operator that can do that, but I couldn't find a way to do it on Windows. It could work like an `exec` command that negates the exit code of the execution.

Sample use case: Stop a commit using `husky` if any file contains string `DO_NOT_COMMIT_THIS`. `"pre-commit": "shx not grep -rq DO_NOT_COMMIT_THIS ."`

**Comment by nfischer** [comment]
I don't think this is currently possible in shx. We support `shx true` and `shx false` (and the OS will support `&&` and `||`), but I believe these are not enough to implement negation.

We could support a `shx --negate` flag to logically negate exit codes. Would you be interested in sending a PR? We would want to check for the flag value here:

https://github.com/shelljs/shx/blob/7e7bf7fe136c923ab6abbc39406fb87444aaf16d/src/shx.js#L92-L93

And then invert the exit code here:

https://github.com/shelljs/shx/blob/7e7bf7fe136c923ab6abbc39406fb87444aaf16d/src/shx.js#L118-L125
